FrameworkSystem/Client/ComponentInstaller.py

```diff
@@ -1963,7 +1965,15 @@ def setupComponent(self, componentType, system, component, extensions,
     if not os.path.lexists(startCompDir):
       gLogger.notice('Creating startup link at', startCompDir)
       mkLink(runitCompDir, startCompDir)
-      time.sleep(10)
```

Comment:
Rather than just sleeping I changed it so it monitors for when `runsvdir` picks up the new link. This makes the tests a faster by a few minutes.